docs/reference/ml/ml-shared.asciidoc

```diff
@@ -951,6 +951,17 @@ Specifies if the tokenization lower case the text sequence when building the
 tokens.
 end::inference-config-nlp-tokenization-bert-do-lower-case[]
 
+tag::inference-config-nlp-tokenization-bert-span[]
+When `truncate` is `none`, this value can be set to indicate if longer text sequences
```

Comment:
The first sentence made me think this was a boolean. Here's a rewrite for your consideration:
```suggestion
When `truncate` is `none`, you can partition longer text sequences
```